### PR TITLE
Fix Slack notification for nightly tests

### PR DESCRIPTION
**Pull request by danielhollas**
Fixed Slack notification and tested that it now works for both nightly and rabbitmq tests.

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -79,7 +76,7 @@ jobs:
       # Always run this step (otherwise it would be skipped if any of the previous steps fail) but only if the
       # `install` or `tests` steps failed, and the `SLACK_WEBHOOK` is available. The latter is not the case for
       # pull requests that come from forks. This is a limitation of secrets on GHA
-      if: always() && (steps.install.outcome == 'Failure' || steps.tests.outcome == 'Failure') && env.SLACK_WEBHOOK != null
+      if: always() && (steps.install.outcome == 'failure' || steps.tests.outcome == 'failure') && env.SLACK_WEBHOOK != null
```
Comment (sphuber) [inline .github/workflows/nightly.yml:79]: Is this essentially the fix?

**Comment by sphuber** [comment]
@danielhollas why are the Docker builds failing? It doesn't seem related to the changes necessarily but the last few runs in other PRs they probably got skipped so we didn't notice? It seems the services in the container are not up again, or at least the health check is timing out. Saw this before some weeks ago and I believe the timeout was increased, as it wasn't clear why the startup would take so long or why they might not be running properly

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (danielhollas) [reply .github/workflows/nightly.yml:79]: Yep!

**Comment by danielhollas** [comment]
I don't know why they are failing. I've tested the build images locally and they work. But they are failing on main branch as well. I'll have to have a closer look, but it should be unrelated to this PR.

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (danielhollas) [reply .github/workflows/nightly.yml:79]: Also the steps in rabbitmq-tests needed ids.

**Pull request merged**